docs/events.md

```diff
@@ -4,6 +4,20 @@ Events that TrenchBoot appeared on.
 
 ## Worldwide conferences
 
+* Xen Project Summit 2024
+    * [Slides](https://static.sched.com/hosted_files/xenprojectsummit2024/59/trenchboot_in_xen.pdf)
+* FOSDEM 2024
+    * [Video](https://video.fosdem.org/2024/aw1126/fosdem-2024-3724-trenchboot-project-status-update.av1.webm)
+    * [Slides DRTM status](https://fosdem.org/2024/events/attachments/fosdem-2024-3724-trenchboot-project-status-update/slides/22916/DRTM-presentation-FOSDEM_Crm6zWi.pdf)
+    * [Slides AEM status](https://fosdem.org/2024/events/attachments/fosdem-2024-3724-trenchboot-project-status-update/slides/22916/TrenchBoot_AEM_Project_Status_P8H1V5D.pdf)
```

Comment:
a/a

Reply:
There are two parts of this presentation - and two sets of slides. Would this deviation from the rest of the document be acceptable?

https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8R10-R16

Reply:
Yes